lib/db/mysql.js

```diff
@@ -701,6 +701,37 @@ module.exports = function (log, error) {
     return this.write(UPDATE_LOCALE, [data.locale, uid])
   }
 
+  var CREATE_UNBLOCK_CODE = 'CALL createUnblockCode_1(?, ?, ?)'
+
+  MySql.prototype.createUnblockCode = function (uid, code) {
+    code = crypto.createHash('sha256').update(uid).update(code).digest()
```

Comment:
It may be worth a short vagueish comment here about "For security, don't store the raw code", otherwise future us might be tempted to simplify this away.